### ISSUE TITLE
Pound sterling £ symbol not displaying correctly

### ISSUE THREAD

**Issue by JakubED** [haozhu233/kableExtra#481]
Hi,

When I include the £ sterling symbol in text when using knitr and kableExtra this displays as <U+00A3> in the output. 

Is there any way I can get £ symbol to display with knitr and kableExtra?

Thanks in advance for any help.

I am using R 3.6.0 and package versions knitr_1.29 and kableExtra_1.1.0. 

Example below:
  
```{r setup, include=FALSE}
knitr::opts_chunk$set(echo=FALSE, warning=FALSE, message=FALSE)

library(rmarkdown)
library(knitr)
library(tidyverse)
library(kableExtra)
library(xtable)

```

```{r make a table, results='asis'}

# input metric
MI_input1 <- data.frame(cbind(c("£15.5m",""),c("£32.3m","157%"),c("","")))


# create MI table
MI_table1 <- kableExtra::kable(MI_input1, col.names = NULL, align = "c") %>%
  kable_styling("bordered", full_width = F) %>%
  pack_rows("Value in £", 1, 2, label_row_css = "text-align: center") %>% 
  column_spec(1:3, width = "1.5cm") %>% 
  row_spec(1:2, align = "c")

MI_table1

```

**Comment by haozhu233** [comment]
Emm, I can't reproduce this behavior. The pound symbol displays correctly on my computer... Can you check if it works without using kableExtra (just use knitr::kable())

**Comment by haozhu233** [comment]
@JakubED This issue might have been fixed in #440.
